set.mm

```diff
@@ -501732,6 +501732,34 @@ have become an indirect lemma of the theorem in question (i.e. a lemma
       FUEBJNPQPRS $.
   $}
 
+  $( The size of a set function is equal to the size of its domain.
+     (Contributed by BTernaryTau, 30-Sep-2023.) $)
+  hashfundm $p |- ( ( F e. V /\ Fun F ) -> ( # ` F ) = ( # ` dom F ) ) $=
+    ( cfn wcel wfun wa chash cfv cdm wceq wi hashfun biimpd adantld wn w3a cpnf
+    hashinf 3adant2 cvv wb fundmfibi notbid adantl dmexg sylan ex adantr sylbid
+    3impia eqtr4d 3comr 3expib pm2.61i ) ACDZABDZAEZFZAGHZAIZGHZJZKUOUQVBUPUOUQ
+    VBALMNUOOZUPUQVBUPUQVCVBUPUQVCPUSQVAUPVCUSQJUQABRSUPUQVCVAQJZURVCUTCDZOZVDU
+    QVCVFUAUPUQUOVEAUBUCUDUPVFVDKUQUPVFVDUPUTTDVFVDABUEUTTRUFUGUHUIUJUKULUMUN
+    $.
```

Comment:
Nice, I think this is a clear improvement compared to [`~hashfun`](https://us.metamath.org/mpeuni/hashfun.html) which only covers the finite case.

Reply:
Thanks!